book/src/help_vc.md

```diff
@@ -10,6 +10,8 @@ FLAGS:
         --builder-proposals
             If this flag is set, Lighthouse will query the Beacon Node for only block headers during proposals and will
             sign over headers. Useful for outsourcing execution payload construction during proposals.
+        --prefer-builder-proposals
```

Comment:
Thanks for getting this ready so quick @eserilev 🙏 

My preference is to rename this to `--always-prefer-builder-proposals`. I don't love the "always" prefix, but that's the name of the flag that was removed. I think it would be nice to avoid users needing to remove `always-` from their configs in order to update.

I'd be open to leaving this flag and adding an `--always..` alias with a deprecation warning if we really want to drop the "always". I just think it'll be a bit of a bumpy upgrade if we drop the flag all of a sudden.

Reply:
Oh crap, sorry!